scripts/functions.py

```diff
@@ -323,24 +322,23 @@ def createFuelSet(countries, rnwTechs, mineTechs, csvPath, generateInternational
     #Hydrogen Fuels
     #hy2FuelList = getHY2fuels(countries)
 
-    #Append lists together and write to a csv
+    #Append lists together and write to a dataframe
     #outputFuels = rnwFuelList + minFuelList + elcFuelList + hy2FuelList
     outputFuels = rnwFuelList + minFuelList + elcFuelList
     dfOut = pd.DataFrame(outputFuels, columns=['VALUE'])
-    dfOut.to_csv(csvPath, index=False)
+    return dfOut
 
-def createTechnologySet(countries, techsMaster, mineTechs, rnwTechs, trnTechsCsvPath, outputCsvPath, generateInternational):
-    # PURPOSE: Appends all technology name lists together and writes them to a CSV
+def createTechDataframe(countries, techsMaster, mineTechs, rnwTechs, trnTechsCsvPath, generateInternational):

```

Comment:
Lets be consistent with naming. This one is called createTechDataFrame, while the function above this is called createFuelSet, but their outputs are synonymous. Can you change one or the other to make them similar please, and make sure the rest in the file are similar as well ?